clangFormat/README.md

```diff
@@ -0,0 +1,9 @@
+## Code Style
+The clang format pre-commit hook will execute clang-format on your code after you commit and inform you if there were any style violations.  All files can be found here: [Code Style](https://github.com/Raizlabs/Raizlabs-Cocoa-Style/tree/master/clangFormat)
+
+ - Download the private clang-format binary, copy it into /usr/local/bin/ and chmod +x it.
```

Comment:
I don't know why we have a custom binary, I assumed it was simply to keep from having to build it.  I've asked NickB and will put his answer in the README.